hryhorieva/hw3/src/main/java/entity/ConnectionDataStorage.java

```diff
@@ -0,0 +1,24 @@
+package entity;
+
+public class ConnectionDataStorage {
+    String host = "localhost:3306";
```

Comment:
Better to do it private static final, because this will not change during Runtime